.github/workflows/unit.yml

```diff
@@ -26,3 +31,4 @@ jobs:
           SNOWFLAKE_ACCOUNT: ${{ secrets.SNOWFLAKE_ACCOUNT }}
           SNOWFLAKE_ROLE: ${{ secrets.SNOWFLAKE_ROLE }}
```

Comment:
rm these secrets, since we should be sure they are not being used in unit tests

also rm `test-acceptance-ci` from targets and replace with `test`